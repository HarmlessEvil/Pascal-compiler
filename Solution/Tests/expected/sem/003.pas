var
	x, y, z: Integer;
	t: Real;
	x: Real;

begin
	x := 1.5;
end.