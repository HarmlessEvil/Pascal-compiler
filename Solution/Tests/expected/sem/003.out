Semantic error at (4, 5): Variable x is already defined